app/assets/stylesheets/application.css.scss.erb

```diff
@@ -110,3 +110,22 @@ $image-path: 'img';
   color: black;
   background: yellow;
 }
+
+.compact-definition {
+  margin-top: 0;
+  margin-bottom: 0;
+
+  dt {
+    clear: both;
+    font-weight: bold;
+  }
+
+  dd {
+    margin: 0
+  }
+
+  dd::after {
+    content: '';
+    display: block;
+  }
```

Comment:
Did you decide against inlining the `dt` and `dd`? We might not need this after all.

```suggestion
```